docs/pages/Index.md

```diff
@@ -4,10 +4,10 @@ permalink: /
 ---
 
 <div align="center">
-  <img alt="Gossamer logo" src="/assets/img/gossamer_banner_white.png" width="500" />
+  <img alt="Gossamer logo" style="opacity: 0.9" src="https://chainsafe.github.io/gossamer/assets/img/gossamer_banner_white.png" width="500" />
 </div>
 <div align="center">
-  <p><i><b>Gossamer Docs</b> - The Official Documentation for Gossamer</i></p>
+  <h1 style="font-size: 110%">Gossamer Docs | <i>The Official Documentation for Gossamer</i></h1>
 </div>
```

Comment:
Maybe put font-size into css for h1?

Reply:
ah, I guess I could since all others are h1, but this was a special case because I wanted the h1 tag for seo/search but did not want the default size. Will probably keep as is for now.

Reply:
I updated it to have a class of `title` so `h1` would not be affect but it would clean this up.